<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Neuron Tracing Generator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="one_skel_tab">
       <attribute name="title">
        <string>One tracing</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="0">
           <widget class="QPushButton" name="one_skel_basal_button">
            <property name="text">
             <string>Open ...</string>
            </property>
            <property name="icon">
             <iconset theme="document-open">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="one_skel_api_button">
            <property name="text">
             <string>Open ...</string>
            </property>
            <property name="icon">
             <iconset theme="document-open">
              <normaloff>../trabajo/Mesh Reconstruct data/MeshReconstruct/src</normaloff>../trabajo/Mesh Reconstruct data/MeshReconstruct/src</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="one_api_vrml_input">
            <property name="placeholderText">
             <string>Input - Apical FilamentTracer VRML/WRL file</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="one_skel_swc_button">
            <property name="text">
             <string>Save ...</string>
            </property>
            <property name="icon">
             <iconset theme="document-save">
              <normaloff>../trabajo/Mesh Reconstruct data/MeshReconstruct/src</normaloff>../trabajo/Mesh Reconstruct data/MeshReconstruct/src</iconset>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="one_skel_swc_input">
            <property name="placeholderText">
             <string>Output file</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="one_basales_vrml_input">
            <property name="placeholderText">
             <string>Input - Basal/s FilamentTracer VRML/WRL file/s</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="one_skel_run_button">
          <property name="text">
           <string>Generate tracing</string>
          </property>
          <property name="icon">
           <iconset theme="media-playback-start">
            <normaloff>../trabajo/Mesh Reconstruct data/MeshReconstruct/src</normaloff>../trabajo/Mesh Reconstruct data/MeshReconstruct/src</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="many_skel_tab">
       <attribute name="title">
        <string>Several tracings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <widget class="QLineEdit" name="many_skel_input">
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="placeholderText">
             <string>Folder with several neurons</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="many_swc_input">
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="placeholderText">
             <string>Folder to save traced neurons</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="many_skel_button">
            <property name="text">
             <string>Open Folder...</string>
            </property>
            <property name="icon">
             <iconset theme="document-open">
              <normaloff>../trabajo/Mesh Reconstruct data/MeshReconstruct/src</normaloff>../trabajo/Mesh Reconstruct data/MeshReconstruct/src</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="many_swc_button">
            <property name="text">
             <string>Save Folder...</string>
            </property>
            <property name="icon">
             <iconset theme="document-save">
              <normaloff>../trabajo/Mesh Reconstruct data/MeshReconstruct/src</normaloff>../trabajo/Mesh Reconstruct data/MeshReconstruct/src</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="many_vrml_run_button">
          <property name="text">
           <string>Generate tracings</string>
          </property>
          <property name="icon">
           <iconset theme="media-playback-start">
            <normaloff>../trabajo/Mesh Reconstruct data/MeshReconstruct/src</normaloff>../trabajo/Mesh Reconstruct data/MeshReconstruct/src</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>40</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="outputFormatLabel">
          <property name="text">
           <string>Output Format</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="outputFormatComboBox">
          <item>
           <property name="text">
            <string extracomment=".dat">NeurolucidaDAT</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>NeurolucidaASC</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>NeurolucidaXML</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>NeurolucidaDATClassic</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>NeurolucidaASCClassic</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SWC</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item row="0" column="1">
         <widget class="QSpinBox" name="connectionThresholdSpinBox">
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="connectionThresholdLabel">
          <property name="text">
           <string>Connection Threshold</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>579</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
